impeller/renderer/backend/vulkan/yuv_conversion_vk.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/renderer/backend/vulkan/yuv_conversion_vk.h"
+
+#include "flutter/fml/hash_combine.h"
+#include "impeller/base/validation.h"
+#include "impeller/renderer/backend/vulkan/device_holder.h"
+#include "impeller/renderer/backend/vulkan/sampler_vk.h"
+
+namespace impeller {
+
+YUVConversionVK::YUVConversionVK(const vk::Device& device,
+                                 YUVConversionDescriptorVK chain)
```

Comment:
linter claims this should be a reference

Reply:
Done.